packages/mobile/src/screens/profile-screen/ProfileInfo.tsx

```diff
@@ -79,9 +77,9 @@ type ProfileInfoProps = {
 
 export const ProfileInfo = (props: ProfileInfoProps) => {
   const { onFollow } = props
+  const { params } = useRoute<'Profile'>()
+  const isOwner = params.handle === 'accountUser'
```

Comment:
if this is the best way to get the `isOwner` value throughout the app maybe we should move this to a shared hook?